components/segment-bridge/base/serviceaccount.yaml

```diff
@@ -29,9 +29,99 @@ roleRef:
 apiVersion: v1
 kind: Secret
 metadata:
-  name: segment-bridge
+  name: segment-bridge-member
   namespace: segment-bridge
   annotations:
-    kubernetes.io/service-account.name: segment-bridge
+    kubernetes.io/service-account.name: segment-bridge-member-sa
 type: kubernetes.io/service-account-token
-
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: segment-bridge-read-member-sa-secret
+  namespace: segment-bridge
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  resourceNames: ["segment-bridge-member"]
+  verbs: ["get"]
+--- 
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: segment-bridge-read-member-sa-secret
+  namespace: segment-bridge
+subjects:
+  - kind: User
+    name: Omeramsc
+  - kind: User
+    name: ifireball
+roleRef:
+  kind: Role
+  name: segment-bridge-read-member-sa-secret
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: segment-bridge-read-access
+  namespace: toolchain-host-operator
```

Comment:
This role should not be here. I would put it into components/authentication. CRs should not be created in namespaces which are not managed by the component.

Reply:
In that case I should put all the `namespace: toolchain-host-operator` CRs there, right?